simdev/module/github/github_api_wrapper.py

```diff
@@ -0,0 +1,209 @@
+import json
+import logging
+import random
+from typing import Any, Dict, List, Set, Union
+
+import requests as requests
```

Comment:
import order

Reply:
What should be different?

Reply:
you have 2 requests - there should be an option to sort imports like these together
![Screenshot 2023-06-14 225926](https://github.com/HSE-JetBrains-department/2023_similar_dev_search_akhundov/assets/2606702/b106c78e-c1d8-4762-9095-e51a3ca6d666)